packages/eslint-plugin/package.json

```diff
@@ -31,12 +31,14 @@
 	],
 	"main": "index.js",
 	"dependencies": {
+		"@typescript-eslint/eslint-plugin": "^4.8.2",
+		"@typescript-eslint/parser": "^4.8.2",
 		"@wordpress/prettier-config": "file:../prettier-config",
 		"babel-eslint": "^10.1.0",
 		"cosmiconfig": "^7.0.0",
 		"eslint-config-prettier": "^6.10.1",
 		"eslint-plugin-jest": "^23.8.2",
-		"eslint-plugin-jsdoc": "^30.2.2",
+		"eslint-plugin-jsdoc": "^30.3.1",
```

Comment:
The lockfile was updating in a way that broke when doing a simple `npm install` with this update dependencies. `eslint-plugin-jsdoc` was moving deeper and apparently couldn't be found when necessary (from scripts package?).

I manually changed the lockfile to keep the updated dependency in its current location, it seems to be stable.

Reply:
It should be all good after 3096429906baf3a001669c59e48527d9ea7a5fe2.